src/components/1/SelectedArticle.jsx

```diff
@@ -0,0 +1,132 @@
+import { FaComment } from "react-icons/fa";
+import { IoChatbubbleOutline } from "react-icons/io5";
+import { FaRegHeart } from "react-icons/fa";
+import styles from "./SelectedArticle.module.css";
+import { lineSpinner } from "ldrs";
+import { useState, useEffect } from "react";
+import { fetchArticleById, fetchCommentsByArticleId, patchVote } from "../../../utils/api.js";
+import { Link } from "react-router-dom";
+import { IoHeartDislikeOutline } from "react-icons/io5";
+import { AiOutlineDislike, AiOutlineLike} from "react-icons/ai";
+import ErrorPage from '../0/ErrorPage';
+
+lineSpinner.register();
+
+const SelectedArticle = ({ article_id}) => {
+
+  const [isOpen, setIsOpen] = useState(false)
+  const [selectedArticle, setSelectedArticle] = useState({});
+  const [isLoading, setIsLoading] = useState(true);
+  const [comments, setComments] = useState([]);
+  const [voteChange, setVoteChange] = useState(0);
+  const [error, setErr] = useState(null)
+  
+  const toggleOpen = () =>{
+    setIsOpen((currOpen)=>!currOpen)
+  }
+  
+  useEffect(() => {
+    setIsLoading(true);
+    Promise.all([fetchArticleById(article_id),fetchCommentsByArticleId(article_id)])
+    .then(([article,comments]) => {
+      setIsLoading(false);
+      setSelectedArticle(article)
+      setComments(comments)
+    })
+    .catch((err)=>{
+      setErr({err})
+    })
+  }, [article_id]);
```

Comment:
I believe this array could be empty as the article id isn't likely to change dynamically, rather when you navigate to another page.